src/disco/containers/DiscoPane.js

```diff
@@ -172,7 +173,12 @@ export class DiscoPaneBase extends React.Component {
 
 export function loadedAddons(state) {
   return state.discoResults.map(
-    (result) => ({ ...result, ...state.addons[result.addon] })
+    (result) => {
+      return {
+        ...result,
+        ...getAddonByGUID(state, result.addon),
```

Comment:
`normalizer` makes this kind of code so hard to understand. You can see how `result.addon` is mapped here: https://github.com/willdurand/addons-frontend/blob/8b810ba3f9d18f729217144c38f6b6ae909538ce/src/disco/api.js#L7

Reply:
Yes, that's what I figured out. I did not want to rewrite everything but I guess adding some comments would be useful.